src/app/_hooks/useBodyScrollLock.js

```diff
@@ -1,25 +1,31 @@
-import { useEffect, useMemo } from 'react';
+import { useEffect } from 'react';
+
+export function useBodyScrollLock(locked) {
+  // const axes = useMemo(
+  //   () => ({
+  //     x: 'overflow-x-hidden',
+  //     y: 'overflow-y-hidden',
+  //     xy: 'overflow-hidden',
+  //   }),
+  //   [],
+  // );
```

Comment:
Uncomment it and use, but you can move it outside of the hook, no need to wrap it in useMemo

Reply:
It looks like this useMemo should be there inside the hook. What do you think?

![image](https://github.com/keenethics/mayak/assets/103147058/0d016d86-4380-4424-97c0-02b65c2ad48a)